tests/network/test_fail_request.py

```diff
@@ -201,8 +206,40 @@ async def test_fail_request_with_auth_required_phase(
             }
         })
 
-    event_response = await wait_for_event(websocket, "cdp.Fetch.requestPaused")
-    assert event_response == {
+    network_event_response = await wait_for_event(websocket,
+                                                  "network.beforeRequestSent")
+    assert network_event_response == {
+        "method": "network.beforeRequestSent",
+        "params": {
+            "context": context_id,
+            "initiator": {
+                "type": "other",
+            },
+            "isBlocked": not exception_and_response_expected,
+            "navigation": ANY_STR,
+            "redirectCount": 0,
+            "request": {
+                "request": ANY_STR,
+                "url": auth_required_url,
+                "method": "GET",
+                "headers": ANY_LIST,
+                "cookies": [],
+                "headersSize": -1,
+                "bodySize": 0,
+                "timings": ANY_DICT,
+            },
+            "timestamp": ANY_TIMESTAMP,
+        },
+        "type": "event",
+    }
+    network_id_from_bidi_event = network_event_response["params"]["request"][
```

Comment:
Same here. You can get network id from navigation result

Reply:
Ditto: Blocked until #1332 is fixed.